RomanASP.py

```diff
@@ -104,12 +118,7 @@ def main():
     fetch_SED = config['fetch_SED']
     makecontourGrid = config['makecontourGrid']
 
-    args = parser.parse_args()
-    band = args.band
-    SNID = args.SNID
-    testnum = args.testnum
-    detim = args.detim
-    output_path = args.output_path
+
```

Comment:
Consider a habit change so that when you delete things you make sure that the number of blank lines left behind is reasonable.  As a starter, delete the entire section rather than replacing it with a blank line.